bears/general/AnnotationBear.py

```diff
@@ -58,7 +62,10 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
             yield Result(self, str(e), severity=RESULT_SEVERITY.MAJOR,
                          affected_code=(e.code,))
 
-        content = {'strings': string_ranges, 'comments': comment_ranges}
+        content = {'singleline strings': ranges[0],
+                   'multiline strings': ranges[1],
+                   'singleline comments': ranges[2],
+                   'multiline comments': ranges[3]}
```

Comment:
Again :) Why not just create a `Ranges` class to hold that information?

Reply:
@sils how does this class help?
This is a class for ranges.
While the return value is 
{key:(range1,range2,..),key2:(),..} 
Are you proposing to make a class having attributes {singleline strings, multiline strings,..} ?

Reply:
@userzimmermann 
1. regarding making new range class - It is being used only once. I don't see the effectiveness of this new range class.
2. regarding creating delimiter class - it is also being passed only once to the `find_annotation_ranges` method.
In other methods, we are sending the delimiter range.
3. regarding using a loop because we are recalculating the same thing 4 times in `find_annotation_ranges` method.
- Actually no, we are not doing the same thing 4 times, we are passing different variables in `get_range_end_position`, and different dicts are being used to store. If we put these things in a loop, It would be like

earlier - 
```
//do stuff1
//do stuff2
//do stuff3
//do stuff4
```

using loop - 

```
for i in range(4):
  if(i==0):
     //do stuff1
  if(i==1):
     //do stuff2
so on..
```

I don't know how does it help.

Reply:
the way you're using this seems more like an object to me, I feel like it'd be way clearer if we just had a simple 6 line class  like that

Reply:
We can make a class with attributes storing tuple of ranges, that's why I thought dict would be better.

Reply:
I'm not sure I get you fully, I'd prefer a class as it's a much clearer and better defined API as "some dict with those keys"